profile/README.md

```diff
@@ -15,7 +15,5 @@ Learn more about [Scayles’s architecture](https://scayle.dev/en/dev/getting-st
 
 ## Other channels
 
-- [GitHub Issues](https://github.com/scayle/storefront-api-php-sdk/issues)
```

Comment:
I'd remove this as it's a specific repository issue tracker and shouldn't be mentioned on the main page of the Organization